bin/import-diff

```diff
@@ -1,6 +1,5 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Why are you changing `bash` into `sh`? Other `import-*` scripts uses `bash`, wouldn't be better to unify this, instead of making differences?

Reply:
If we don't any extra features of bash, we may want to eventually switch to alpine base, which doesn't have bash (currently the `import-data` image is the only alpine-based one). No biggie here, up to you.